### PR TITLE
refactor: #CO-1721 improve servicemanager and services

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
Improve servicemanager and services. Refactor to respect convention.

## Issue ticket number and link
[CO-1721](https://jira.support-ent.fr/browse/CO-1721)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/java/fr/openent/supportpivot/services/HttpClientService.java [hunk 1]
```diff
@@ -1,48 +1,14 @@
 package fr.openent.supportpivot.services;
```
Comment (BastienBenard) [inline src/main/java/fr/openent/supportpivot/services/HttpClientService.java:1]: HttpClientService a était renommé en HttpClientServiceImpl. Puis j'ai crée HttpClientService comme interface pour HttpClientServiceImpl

File: src/main/java/fr/openent/supportpivot/services/MongoService.java [hunk 2]
```diff
@@ -1,64 +1,10 @@
 package fr.openent.supportpivot.services;
```
Comment (BastienBenard) [inline src/main/java/fr/openent/supportpivot/services/MongoService.java:1]: MongoService a était renommé en MongoServiceImpl. Puis j'ai crée MongoService comme interface pour MongoServiceImpl

File: src/main/java/fr/openent/supportpivot/services/impl/HttpClientServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
+package fr.openent.supportpivot.services.impl;
```
Comment (BastienBenard) [inline src/main/java/fr/openent/supportpivot/services/impl/HttpClientServiceImpl.java:1]: Voir commentaire [HttpClientService.java](https://github.com/OPEN-ENT-NG/support-pivot/pull/32/files#diff-3e8a9a37991f4a8d4f52850af1f21498a4e8c50beee6945a1b4e6723d9b234a7)

File: src/main/java/fr/openent/supportpivot/services/impl/MongoServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
+package fr.openent.supportpivot.services.impl;
```
Comment (BastienBenard) [inline src/main/java/fr/openent/supportpivot/services/impl/MongoServiceImpl.java:1]: Voir commentaire [MongoService.java](https://github.com/OPEN-ENT-NG/support-pivot/pull/32/files#diff-5e01d18c5ab066d025b38784c2e48382b36634c130ad6fae2e2bf21411729de6)

**Review by JinDouang — APPROVED** [review]

**Pull request merged**